### ISSUE TITLE
Linux aem and tailon services don't autostart after reboot

### ISSUE THREAD

**Issue by nesreb** [ahmed-musallam/AEM-UP#3]
Ahmed, first thanks very much for laying the groundwork for something that looks to be very helpful.

I'm running the vm on macOS Mojave, with Ansible 2.7.9 (also tried 2.7.10), VirtualBox 6.0.4 (6.0.6 has vbguest plugin install issues), and Vagrant 2.2.4.  The three services you set up (author, publish, and tailon) don't autostart after reboot.  It might be an Ansible bug because there have been reports of this issue in the past with other services (e.g., nginx).  Wanted to verify you see the same issue with a fresh install before I dig any deeper, or if you already have a suggested workaround in place.  Everything is fine after starting the services manually though.

**Comment by ahmed-musallam** [comment]
Yeah this appears to be a bug, the init scripts don't start on reboot. Opened #4 for it. Feel free to make the switch if you'd like. It wont be for a few days before I get to it.

I did notice it was not working before, but missed creating a bug!

A simple work around for now is to tag the tasks that run the systemd services then run ansible with only those tags after a reboot. See: https://docs.ansible.com/ansible/latest/user_guide/playbooks_tags.html
